contracts/src/v0.8/ccip/pools/GHO/UpgradeableTokenPool.sol

```diff
@@ -250,7 +257,8 @@ abstract contract UpgradeableTokenPool is IPool, OwnerIsCreator, IERC165 {
   /// is a permissioned onRamp for the given chain on the Router.
   modifier onlyOnRamp(uint64 remoteChainSelector) {
     if (!isSupportedChain(remoteChainSelector)) revert ChainNotAllowed(remoteChainSelector);
-    if (!(msg.sender == s_router.getOnRamp(remoteChainSelector))) revert CallerIsNotARampOnRouter(msg.sender);
+    if (msg.sender != s_router.getOnRamp(remoteChainSelector) && msg.sender != getProxyPool(remoteChainSelector))
```

Comment:
I thought CCIP code style is to use negatives for these conditionals: if not met that [condition], revert.
Could you validate that's the case? If so, i d change it as the suggestion
```suggestion
    if (!(msg.sender == s_router.getOnRamp(remoteChainSelector) || msg.sender == getProxyPool(remoteChainSelector))
```

Reply:
better to use `{}` if it cannot be in 1 single line

Reply:
i'd looked into that & it's not followed consistently ([ex](https://github.com/aave/ccip/blob/ccip-gho/contracts/src/v0.8/ccip/AggregateRateLimiter.sol#L91), [ex](https://github.com/aave/ccip/blob/ccip-gho/contracts/src/v0.8/ccip/ARM.sol#L283-L286))
but i've pushed the suggestion and moved the pool proxy condition first to short circuit the if to save a bit of gas (saving one staticcall & one warn sload) - [fd10bb](https://github.com/aave/ccip/pull/18/commits/6fd10bb3216bce0221ca51b2025568a0159ecd58)
(it still stays in 1 one haha)